.travis.yml

```diff
@@ -7,7 +7,8 @@ node_js:
 
 notifications:
     email: false
-
+before_install:
+  - npm install -g typings
```

Comment:
oh now i got, i shouldn't add a global dependency as it's installed inside the project with the correct version, thanks :)